Sources/Commands/Options.swift

```diff
@@ -42,4 +42,26 @@ public class ToolOptions {
 }
 
 /// Parser conformance for ColorWrap.
-extension ColorWrap.Mode: StringEnumArgument {}
+extension ColorWrap.Mode: StringEnumArgument {
+    public static var completion: ShellCompletion {
+        return .values([
+            (Auto.description, "auto detect color support"),
+            (Always.description, "always output with color"),
+            (Never.description, "never output with color")
+        ])
+    }
+}
+
+struct PathArgument: ArgumentKind {
+    let path: String
```

Comment:
We discussed that constructing the `AbsolutePath` could take place inside `PathArgument.init`.